tot/datasets/dataset.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from dataclasses import dataclass, field
+from typing import List, Optional, Tuple, Type
```

Comment:
Tuple and type unused
```suggestion
from typing import List, Optional
```